diff -ru dracut-007.orig//modules.d/50plymouth/installkernel dracut-007/modules.d/50plymouth/installkernel
--- dracut-007.orig//modules.d/50plymouth/installkernel	2010-08-09 16:13:51.000000000 +0200
+++ dracut-007/modules.d/50plymouth/installkernel	2010-09-18 21:36:22.048443676 +0200
@@ -3,3 +3,9 @@
 for modname in $(find "$srcmods/kernel/drivers/gpu/drm" -name '*.ko' 2>/dev/null); do
     grep -q drm_crtc_init  $modname && instmods $modname
 done
+# Include uvesafb
+[ -e /sbin/v86d ] && {
+   instmods uvesafb
+   inst /sbin/v86d
+   inst /sbin/testvbe
+}
diff -ru dracut-007.orig//modules.d/50plymouth/plymouth-pretrigger.sh dracut-007/modules.d/50plymouth/plymouth-pretrigger.sh
--- dracut-007.orig//modules.d/50plymouth/plymouth-pretrigger.sh	2010-08-09 16:13:51.000000000 +0200
+++ dracut-007/modules.d/50plymouth/plymouth-pretrigger.sh	2010-09-18 21:12:37.753313417 +0200
@@ -8,6 +8,9 @@
     udevadm trigger --subsystem-match=graphics --subsystem-match=drm --subsystem-match=tty >/dev/null 2>&1
 
     udevadm settle --timeout=30 2>&1 | vinfo
+
+    [ ! -c /dev/fb0 -a -e /sbin/v86d ] && modprobe uvesafb
+
     [ -c /dev/zero ] || mknod -m 0666 /dev/zero c 1 5
     [ -c /dev/tty0 ] || mknod -m 0620 /dev/tty0 c 4 0
     [ -e /dev/systty ] || ln -s tty0 /dev/systty
