diff -ur daemons/dmeventd/plugins/mirror/dmeventd_mirror.c daemons/dmeventd/plugins/mirror/dmeventd_mirror.c
--- daemons/dmeventd/plugins/mirror/dmeventd_mirror.c	2009-06-30 19:39:31.000000000 +0100
+++ daemons/dmeventd/plugins/mirror/dmeventd_mirror.c	2009-11-17 21:01:01.932535235 +0000
@@ -128,6 +128,7 @@
 
 static void _temporary_log_fn(int level, const char *file __attribute((unused)),
 			      int line __attribute((unused)),
+			      int dm_err __attribute((unused)),
 			      const char *format)
 {
 	if (!strncmp(format, "WARNING: ", 9) && (level < 5))
diff -ur daemons/dmeventd/plugins/snapshot/dmeventd_snapshot.c daemons/dmeventd/plugins/snapshot/dmeventd_snapshot.c
--- daemons/dmeventd/plugins/snapshot/dmeventd_snapshot.c	2008-10-13 13:06:30.000000000 +0100
+++ daemons/dmeventd/plugins/snapshot/dmeventd_snapshot.c	2009-11-17 21:01:19.520632278 +0000
@@ -57,6 +57,7 @@
 static void _temporary_log_fn(int level,
 			      const char *file __attribute((unused)),
 			      int line __attribute((unused)),
+			      int dm_err __attribute((unused)),
 			      const char *format)
 {
 	if (!strncmp(format, "WARNING: ", 9) && (level < 5))
diff -ur tools/tools/lvm2cmd.h
--- tools/lvm2cmd.h	2007-08-20 21:55:30.000000000 +0100
+++ tools/lvm2cmd.h	2009-11-17 20:59:29.840323142 +0000
@@ -18,6 +18,7 @@
 
 #ifndef _LVM_LOG_H
 typedef void (*lvm2_log_fn_t) (int level, const char *file, int line,
+				   int dmerr,
 			       const char *message);
 #endif
 
