diff -ru /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//examples/pullcons.cc ./examples/pullcons.cc
--- /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//examples/pullcons.cc	2005-04-13 16:37:08.000000000 +0100
+++ ./examples/pullcons.cc	2011-09-22 16:13:20.567110893 +0100
@@ -140,6 +140,7 @@
 using namespace std;
 #endif
 
+#include <stdio.h>
 #ifdef HAVE_STDLIB_H
 #  include <stdlib.h>
 #endif
diff -ru /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//examples/pullsupp.cc ./examples/pullsupp.cc
--- /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//examples/pullsupp.cc	2005-04-13 16:37:08.000000000 +0100
+++ ./examples/pullsupp.cc	2011-09-22 16:13:39.633654575 +0100
@@ -136,6 +136,7 @@
 using namespace std;
 #endif
 
+#include <stdio.h>
 #ifdef HAVE_STDLIB_H
 #  include <stdlib.h>
 #endif
diff -ru /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//examples/pushcons.cc ./examples/pushcons.cc
--- /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//examples/pushcons.cc	2005-06-28 19:52:37.000000000 +0100
+++ ./examples/pushcons.cc	2011-09-22 16:13:52.860235560 +0100
@@ -151,6 +151,7 @@
 using namespace std;
 #endif
 
+#include <stdio.h>
 #ifdef HAVE_STDLIB_H
 #  include <stdlib.h>
 #endif
diff -ru /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//examples/pushsupp.cc ./examples/pushsupp.cc
--- /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//examples/pushsupp.cc	2005-04-13 16:37:08.000000000 +0100
+++ ./examples/pushsupp.cc	2011-09-22 16:14:07.510140310 +0100
@@ -144,6 +144,7 @@
 using namespace std;
 #endif
 
+#include <stdio.h>
 #ifdef HAVE_STDLIB_H
 #  include <stdlib.h>
 #endif
diff -ru /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//src/main.cc ./src/main.cc
--- /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//src/main.cc	2005-05-11 22:56:09.000000000 +0100
+++ ./src/main.cc	2011-09-22 16:07:05.722731214 +0100
@@ -31,6 +31,7 @@
 #  include "config.h"
 #endif
 
+#include <stdlib.h>
 #ifdef HAVE_GETOPT
 #  include <unistd.h>
 extern char* optarg;
diff -ru /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//src/PersistNode.cc ./src/PersistNode.cc
--- /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//src/PersistNode.cc	2005-04-13 16:37:09.000000000 +0100
+++ ./src/PersistNode.cc	2011-09-22 16:09:50.008426781 +0100
@@ -23,6 +23,7 @@
 
 #include "PersistNode.h"
 
+#include <limits.h>
 #include <stdlib.h>
 #include <stdio.h>
 
diff -ru /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//src/Servant.cc ./src/Servant.cc
--- /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//src/Servant.cc	2005-05-17 22:58:57.000000000 +0100
+++ ./src/Servant.cc	2011-09-22 16:09:43.551799216 +0100
@@ -28,6 +28,7 @@
 #  include <sys/types.h> // getpid
 #endif
 
+#include <stdlib.h>
 #ifdef HAVE_UNISTD_H
 #  include <unistd.h>    // getpid
 #elif defined(HAVE_PROCESS_H)
diff -ru /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//tools/eventc.cc ./tools/eventc.cc
--- /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//tools/eventc.cc	2005-05-11 19:44:31.000000000 +0100
+++ ./tools/eventc.cc	2011-09-22 16:12:00.157622638 +0100
@@ -176,6 +176,7 @@
 using namespace std;
 #endif
 
+#include <stdio.h>
 #ifdef HAVE_STDLIB_H
 #  include <stdlib.h>
 #endif
diff -ru /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//tools/eventf.cc ./tools/eventf.cc
--- /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//tools/eventf.cc	2005-05-11 19:44:31.000000000 +0100
+++ ./tools/eventf.cc	2011-09-22 16:12:22.457481843 +0100
@@ -36,6 +36,7 @@
 #  include "getopt.h"
 #endif
 
+#include <stdio.h>
 #ifdef HAVE_STDLIB_H
 #  include <stdlib.h> // exit()
 #endif
diff -ru /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//tools/rmeventc.cc ./tools/rmeventc.cc
--- /var/tmp/portage/dev-libs/omniEvents-2.6.2/work/omniEvents-2_6_2//tools/rmeventc.cc	2005-05-11 19:44:31.000000000 +0100
+++ ./tools/rmeventc.cc	2011-09-22 16:12:41.950691394 +0100
@@ -36,6 +36,7 @@
 #  include "getopt.h"
 #endif
 
+#include <stdio.h>
 #ifdef HAVE_STDLIB_H
 #  include <stdlib.h> // exit()
 #endif
