diff --git a/bdr_init_copy.c b/bdr_init_copy.c
index 3ab0bce..3e6c8c4 100644
--- a/bdr_init_copy.c
+++ b/bdr_init_copy.c
@@ -1728,7 +1728,7 @@ wait_postmaster_connection(const char *connstr)
 	}
 	else
 	{
-		print_msg(VERBOSITY_VERBOSE, _("\npostmaster started (pid="INT64_FORMAT"), waiting for connection"), pmpid);
+		print_msg(VERBOSITY_VERBOSE, _("\npostmaster started (pid=%ld), waiting for connection"), pmpid);
 	}
 
 	/*
