--- config/Make.common.rules	2015-06-23 16:30:20.000000000 +0100
+++ config/Make.common.rules	2015-07-04 17:45:58.061422073 +0100
@@ -115,15 +115,6 @@
     lp32suffix     = /i386-linux-gnu
 endif
 
-ifeq ($(CPP11),yes)
-    ifneq ($(UNAME),Darwin)
-        cpp11libdirsuffix	= /c++11
-        cpp11libsuffix		= ++11
-        cpp11sonamedir		= ../
-        binsuffix 		:= $(binsuffix)$(cpp11libsuffix)
-    endif
-endif
-
 ifneq ($(findstring MINGW,$(UNAME)),)
     UNAME	                := MINGW
     lp64suffix                  := /x64
